customTests/azure_ib_write_bw_gdr.nhc

```diff
@@ -0,0 +1,65 @@
+#!/bin/bash
+
+# Expected bandwidth > 180 Gbps
+
+IB_WRITE_BW_EXE_PATH=/opt/azurehpc/test/nhc/perftest-4.5/ib_write_bw
+IB_WRITE_BW=`basename $IB_WRITE_BW_EXE_PATH`
+IB_WRITE_BW_DURATION=10
+IB_WRITE_BW_ARGS="-s $(( 1 * 1024 * 1024 )) -D ${IB_WRITE_BW_DURATION} -x 0 -F --report_gbits"
+SLEEP_TIME=5
+
+HOSTNAME=`hostname`
+#NUMA mapping for NDv4(A100)
+GPU_NUMA=( 1 1 0 0 3 3 2 2 )
```

Comment:
Do we want this hardcoded in the test? I guess the alternatives are to have it passable from the command line, or to try every numa for each GPU. Maybe having it passable from the command line would be better? For now this is probably ok given the limited list of skus we are targeting at the moment.

Reply:
Yea, I was going back and forth a bit with this but I settled on making it work with NDv4 (hard coded). Then submit a PR when we have had adequate time to test on NDv5.